### PR TITLE
fix editor cursor location (due to font-family override in SRC)

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -161,8 +161,8 @@
     "build:js": "tsc --p tsconfig.build.json",
     "build:copy-assets": "cd src/lib && find . -name '*.scss' -o -name '*.css' -o -name '*.svg' | cpio -pdm ../../dist/",
     "build:rollup": "rollup -c && cp -r src/umd dist/",
-    "build:docs": "npx styleguidist build",
-    "start-docs": "npx styleguidist server"
+    "build:docs": "yarn run build-css && npx styleguidist build",
```
Comment (jay-hodgson) [inline package.json:164]: build css when building docs/demo site or starting local server.

File: src/demo/style/DemoStyle.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
-$primary-action-color: #c94281;
-$primary-bgcolor-rgba: rgba(201, 66, 129, 0.03);
+$primary-action-color: rgb(64, 123, 160);
+$primary-bgcolor-rgba: rgba(64, 123, 160, 0.03);
```
Comment (jay-hodgson) [inline src/demo/style/DemoStyle.scss:2]: Change from hotpink to nf's current color scheme from design.

File: src/lib/style/base/_base.scss [hunk 3]
```diff
@@ -21,12 +21,9 @@ html {
...
 }
 
-body * {
-  font-family: 'Lato';
-}
-
 body {
   color: $gray-regular;
+  font-family: 'Lato';
 }
```
Comment (jay-hodgson) [inline src/lib/style/base/_base.scss:28]: This is the main fix.  "body *" is a bit too aggressive a selector, it applied to <pre><span class="token string/operator/punctuation/..."></span></pre>, and changed the font family in codemirror.  The way codemirror works (today) is that it has a textarea (with absolute position) overlaying a pre, where the pre is used to show the actual code (with syntax highlighting, etc).  Once we mucked with the style, these two were no longer in sync, and the cursor was no longer on the textfield text that the user wants to edit!

**Review by emmanmills — APPROVED** [review]
Looks good!

File: styleguide.config.js [hunk 4]
```diff
@@ -1,8 +1,9 @@
 const path = require('path')
+const pkg = require('./package.json');
 const ignoreComponents = ['**/*.Mobile.tsx','**/*.Desktop.tsx']
 
 module.exports = {
-    title: 'Synapse React Client Component Library',
+    title: `Synapse React Components v${pkg.version}`,
```
Comment (jay-hodgson) [inline styleguide.config.js:6]: added current package version

File: styleguide.config.js [hunk 5]
```diff
@@ -36,14 +37,16 @@ module.exports = {
...
-    // This works when running the local server, but not when built.
-    // styles: {
-    //     StyleGuide: {
-    //         content: {
-    //             maxWidth: 1300
-    //         }
-    //     }
-    // },
+    theme: {
+        maxWidth: 1300,
```
Comment (jay-hodgson) [inline styleguide.config.js:41]: found out how to properly customize the maxWidth (of the content)!

Comment (emmanmills) [reply styleguide.config.js:41]: Oops! Didn't see this. Do you need to add a unit after 1300?

**Pull request merged**